src/website/interface.ts

```diff
@@ -39,6 +49,7 @@ export interface WebsiteTournament {
 	url: string;
 	players: WebsitePlayer[];
 	rounds: number;
+	tieBreaks: ChallongeTieBreaker[];
```

Comment:
This very much can be `null` before the tournament is started, even though an ordering is shown in the UI.

Reply:
Uh oh. In that case it's the challonge default?